src/main/java/racingcar/service/RaceWinnerService.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.service;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import racingcar.model.CarName;
+import racingcar.model.CarScores;
+import racingcar.model.Distance;
+import racingcar.view.OutputView;
+
+public class RaceWinnerService {
+    private static final int INIT_MAX_TO_ZERO = 0;
+
+    public static void findLastWinner(CarScores carScores) {
+        Distance maxDistance = new Distance(INIT_MAX_TO_ZERO);
+        for (Map.Entry<CarName, Distance> nowCarScore : carScores.getCarScores().entrySet()) {
+            Distance nowDistance = nowCarScore.getValue();
+            maxDistance = getNewMax(maxDistance.getDistance(), nowDistance.getDistance());
+        }
+
+        List<String> lastWinners = getLastWinners(carScores.getCarScores(), maxDistance);
+        OutputView.printFinalWinner(lastWinners);
+    }
+
+    private static Distance getNewMax(int max, int nowScore) {
+        return new Distance(Math.max(nowScore, max));
+    }
+
+    private static List<String> getLastWinners(LinkedHashMap<CarName, Distance> nowCarScores, Distance maxDistance) {
+        return nowCarScores.entrySet().stream()
+                .filter(entry -> entry.getValue().equals(maxDistance))
+                .map(Map.Entry::getKey)
+                .map(CarName::getCarName)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .collect()
                .toList();
```
이렇게 바꾸는게 더 좋을 것 같습니다! 저도 이번에 리뷰 받으면서 새로 알았는데 ```toList()```가 불변성을 보장해줍니다.
[Stream.toList() VS Stream.collect(toList()](https://binux.tistory.com/146)

Reply:
`.collect(Collectors.toList())`를 `.toList()`로 바꾸는 부분 말씀하신 게 맞으실까요?!
좋은 정보 감사합니다 😊

Reply:
넵 맞습니다!